ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -838,7 +841,7 @@ static PrjFS_Result HandleRecursivelyEnumerateDirectoryRequest(const MessageHead
         
         CombinePaths(s_virtualizationRootFullPath.c_str(), directoryRelativePath.c_str(), path);
     
-        PrjFS_Result result = HandleEnumerateDirectoryRequest(request, directoryRelativePath.c_str());
+        PrjFS_Result result = HandleEnumerateDirectoryRequest(request, path, directoryRelativePath.c_str());
```

Comment:
I realize this is existing but directoryAbsolutePath would be clearer than 'path'